homeassistant/components/sensibo/climate.py

```diff
@@ -145,6 +147,11 @@ def get_features(self) -> int:
                 features |= FIELD_TO_FLAG[key]
         return features
 
+    @property
+    def extra_state_attributes(self) -> dict[str, Any]:
+        """Return additional attributes."""
+        return {"temperature_list": self.coordinator.data[self.unique_id]["temp_list"]}
```

Comment:
Generally we want sensor entities instead of `extra_state_attributes`, but since the goal is to target a patch release, lets remove this for now.

Reply:
I think this would make more sense to have as an attribute on the climate entity as it's a list of temperatures so the users can see the option that is available to set temp.
Would seem a bit strange to me to have a list of temps as a sensor?

Will remove it for the patch but would like to get feedback on it as I would think it should be put back.

Reply:
We have max and min temperature and step size attributes that can be used by the frontend to inform the user about the settings. Isn't that enough?

Reply:
The "problem" is that some devices has temperatures such as 10, 16, 17 etc. so the stepping is not necessarily the same all the time for the whole list. Therefore I thought it would make sense to present it somewhere for the user to see what options is available and for me would make more sense as an attribute on the climate entity (similar to presenting the swing options) rather than as a sensor or something else.

Reply:
I don't think that a separate list of available temperatures in the state attributes is user friendly for the frontend as it doesn't affect how the temperature selection is shown.

We shouldn't add it to the state machine for that purpose in my opinion.

Reply:
Fine. I'm not going to argue on it as it perhaps was a more nice to have rather than must and probably is for a rather small portion of the users with some specific heaters/ac's.